components/trace/appmap/event/index.test.mjs

```diff
@@ -15,163 +15,138 @@ const { createConfiguration } = await buildTestComponentAsync(
   "configuration",
   "test",
 );
-const { compileEventTrace } = Event(dependencies);
+const { digestEventTrace } = Event(dependencies);
 
-{
-  const classmap = createClassmap(createConfiguration("file:///home"));
-  addClassmapSource(classmap, {
-    url: "file:///home/filename.js",
-    content: "function f (x) {}",
-    inline: false,
-    exclude: [
+const makeEvent = (site, tab, payload) => ({
+  type: "event",
+  site,
+  tab,
```

Comment:
Well it is not depth neither. Basically it is a number that identifies a pair event being a begin/end or a before/after. I agree that tab is not the best name but index was just too generic. Where would you like these comments? In `build/schema/schema.yml`? In general, I try to avoid comments as they rot away. But maybe this is just because I have not collaborated enough in the past.

Reply:
I understand your concern about rotting comments -- it's important to keep them up to date, so you don't want to have too many (and definitely don't repeat yourself). But it's equally important to have some whenever something is not obvious :) Any place is fine, preferably as authoritative as possible, so it'd be difficult to let it rot :)

Reply:
here they are 7ad24a4e8a8c3388fb0860dab9890098d8e5551c